htlcswitch/switch.go

```diff
@@ -1378,8 +1393,17 @@ func (s *Switch) htlcForwarder() {
 	fwdEventTicker := time.NewTicker(15 * time.Second)
 	defer fwdEventTicker.Stop()
 
+out:
 	for {
 		select {
+		// A new block has arrived so we'll update our best height.
+		case blockEpoch, ok := <-s.blockEpochs.Epochs:
+			if !ok {
+				break out
+			}
+
+			atomic.StoreUint32(&s.bestHeight, uint32(blockEpoch.Height))
```

Comment:
Carrying over a comment here to recommend a method is added so the link don't need to directly access this attribute.

Reply:
Fixed.